riscv/src/elf/mod.rs

```diff
@@ -859,15 +859,15 @@ impl TwoOrOneMapper<MaybeInstruction, HighLevelInsn> for InstructionLifter<'_> {
 
                         HighLevelInsn { op, args, loc }
                     }
-                    // l{b|h|w} rd, symbol
+                    // l{b|h|w}[u] rd, symbol
                     Ins {
                         opc: l_op,
                         rd: Some(rd_l),
                         rs1: Some(rs1_l),
                         rs2: None,
                         imm: Some(lo),
                         ..
-                    } if matches!(l_op, Op::LB | Op::LH | Op::LW)
+                    } if matches!(l_op, Op::LB | Op::LH | Op::LW | Op::LBU | Op::LHU | Op::LWU)
```

Comment:
I am not sure `LWU` should be included, as it is a RV64-only instruction and all the rest of the code assumes RV32.

Reply:
it's definitely rv32, my compiler generates it.

Reply:
It is probably either `LBU` or `LHU`, because our codegen doesn't implement `LWU`, either, so this fix wouldn't be enough.

`LWU` is defined in section [4.3. Load and Store Instructions](https://riscv.github.io/riscv-isa-manual/snapshot/unprivileged/#_load_and_store_instructions) of the spec, under [4. RV64I Base Integer Instruction Set](https://riscv.github.io/riscv-isa-manual/snapshot/unprivileged/#rv64).

Reply:
totally misread your comment, yeah that's right, just B and H 👍

Reply:
```suggestion
                    } if matches!(l_op, Op::LB | Op::LH | Op::LW | Op::LBU | Op::LHU)
```